.github/workflows/main.yml

```diff
@@ -4,11 +4,10 @@ name: CI
 
 # Controls when the action will run. 
 on:
-  # Triggers the workflow on push or pull request events but only for the dev branch
+  # Triggers the workflow on push to the dev branch
   push:
     branches: [ dev ]
   pull_request:
-    branches: [ dev ]
```

Comment:
Can you think of a sanity check / unit test that we can still do here?

Reply:
We don't need to consider it in the scope of this PR, but some sort of check would be good.  Perhaps https://github.com/jimhester/lintr with check syntax checking to start?

Reply:
Sounds good, I will add an issue for that.

Reply:
see also https://devtools.r-lib.org/reference/check.html